api/agentpb/agent.proto

```diff
@@ -414,6 +414,18 @@ message ParseDefaultsFileResponse {
   string socket = 6;
 }
 
+// AgentLogsRequest is an ServerMessage asking logs by Agent ID.
+message AgentLogsRequest {
+  string agent_id = 1;
+  uint32 limit = 2;
+}
+
+// AgentLogsResponse is an AgentMessage containing logs by Agent ID.
+message AgentLogsResponse {
+  repeated string logs = 1;
+  uint32 agent_config_log_lines_count = 2;
```

Comment:
I wonder why we need this information on the FE side.

Reply:
@BupycHuk i think "agent_config_log_lines_count" will be good for troubleshooting or for understand "how many logs we can receive from API"